src/plugins/banner/public/components/global_banner.tsx

```diff
@@ -0,0 +1,167 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Any modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import React, { Fragment, useEffect, useState, Suspense, useRef } from 'react';
+import { EuiCallOut, EuiLoadingSpinner } from '@elastic/eui';
+import { BannerConfig } from '../../common';
+import { BannerService } from '../services/banner_service';
+import { bannerServiceInstance } from '../services/banner_service_instance';
+
+const ReactMarkdownLazy = React.lazy(() => import('react-markdown'));
+
+// Custom link renderer for markdown links to open in new tab
+const LinkRenderer = (props: any) => {
```

Comment:
Nit: can this be extracted into a separate components file like `link_renderer.tsx`? There seems to be multiple components when it should just contain the `GlobalBanner`